### ISSUE TITLE
Generic description field on recipes

### ISSUE THREAD

**Issue by incaseoftrouble** [vabene1111/recipes#268]
Hi,

a small suggestion: Add a description field to the recipe, displayed alongside / below the general information (picture & ingredients).

Idea for the layout:
```
+----------------+-----------------+
|                |                 |
|  Ingredients   |   Description   |
|                |                 |
+----------------+-----------------+
```
or
```
+----------------+-----------------+
|                |                 |
|    Picture     |   Ingredients   |
|                |                 |
+----------------+-----------------+
|                                  |
|            Description           |
|                                  |
+----------------------------------+
```

**Comment by vabene1111** [comment]
yea i was thinking about something like this too. Could also be displayed in meal plan and search results.  Probably limited to a few character (~ 100-200)

**Comment by smilerz** [comment]
This was resolved by #497

**Comment by vabene1111** [comment]
correct, i completely forgot about this issue, thanks !
